hl13/code/modules/projectiles/projectile/bullets.dm

```diff
@@ -13,3 +13,11 @@
 	wound_bonus = 0
 	wound_falloff_tile = -5
 	embed_falloff_tile = -3
+
+/obj/projectile/bullet/hl13/pulse
+	name = "Dark Energy Pulse"
+	icon_state = "pulse_bullet"
+	damage = 8
+	embedding = list(embed_chance=15, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
```

Comment:
pulse shouldnt have fall_chance or at least should be really low (since its basically a beam of particles)

Reply:
fall chance is at a zero